samples/Tests/CustomParsingTestUsingProgressBar.json

```diff
@@ -0,0 +1,31 @@
+{
+    "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
+    "type": "AdaptiveCard",
+    "version": "1.0",
+    "body": [
+                {
+                    "type": "TextBlock",
+                    "text": "Custom parsing allows a custom type",
+                    "wrap": true,
+                    "horizontalAlignment": "center"
+                },
+                {
+                    "type": "TextBlock",
+                    "text": "**Progress Bar**",
+                    "horizontalAlignment": "center"
+                },
+                {
+                    "type": "CustomIOSDelegate",
+                    "payload": [
```

Comment:
Looks good. I see customer has freedom to populate any data to consume in its own custom renderer.

Reply:
that's is correct

---
In reply to: [172233766](https://github.com/Microsoft/AdaptiveCards/pull/1211#discussion_r172233766) [](ancestors = 172233766)